### ISSUE TITLE
Console error if datagrid no longer present

### ISSUE THREAD

**Issue by kingy04** [CroudTech/Croud-Style-Guide#74]
When using the datagrid in a sidebar or modal a console error can occur when using this component as when the refresh method is called the datagrid is no longer present.

We could check if `this.$refs.vuetable.$el` exists before calling this code to prevent this? @BrockReece?

https://github.com/CroudTech/Croud-Style-Guide/blob/aac257fddcd6d8fa754fa835ffa13fc324384cd9/src/components/shared/Datatable.vue#L180

**Comment by kingy04** [comment]
`Uncaught TypeError: Cannot read property 'refresh' of undefined`

**Comment by BrockReece** [comment]
Yep, checking if `this.$refs.vuetable` exists first makes sense.

**Comment by BrockReece** [comment]
Can you give me some steps to reproduce this issue so I can test the PR?

**Comment by kingy04** [comment]
if you add a file to open the sidebar, and click finish, you should see a console error when the sidebar closes.

**Comment by BrockReece** [comment]
Ok, I tried this but I am getting an error when uploading a file on my local dev env, looks to be building the url incorrectly.
https://github.com/CroudTech/cc-vuejs-monorepo/issues/130
